src/main/resources/com/dabsquared/gitlabjenkins/GitLabPushTrigger/config.jelly

```diff
@@ -18,6 +18,9 @@
     <f:entry title="Add note with build status on merge requests" field="addNoteOnMergeRequest">
       <f:checkbox default="true" />
     </f:entry>
+    <f:entry title="Use GitLab CI features (GitLab 8.1 required!)" field="addCiMessage">
```

Comment:
I'll check to the code tomorrow, and add any missing things which I overlooked.. It's still backward compatability testable, because the code still defaults to false :)